model/database/spans.yaml

```diff
@@ -94,14 +94,103 @@ groups:
       - ref: db.namespace
         sampling_relevant: true
         brief: The name of the database, fully qualified within the server address and port.
-        note:
+        note: |
           When connecting to a default instance, `db.namespace` SHOULD be set to the name of
           the database. When connecting to a [named instance](https://learn.microsoft.com/sql/connect/jdbc/building-the-connection-url#named-and-multiple-sql-server-instances),
           `db.namespace` SHOULD be set to the combination of instance and database name following the `{instance_name}.{database_name}` pattern.
 
-          For commands that switch the database, this SHOULD be set to the target database (even if the command fails).
+          The current database may change during the lifetime of a connection, e.g. from executing `USE <database>`.
+
+          If instrumentation is unable to capture the current database without causing an additional query to be executed
+          (e.g. `SELECT DB_NAME()`), then it is RECOMMENDED to set `db.namespace` to the database name provided at connection time
+          instead of not capturing any value for `db.namespace`.
+
+          Instrumentation SHOULD document if `db.namespace` only reflects the database name provided during at connection time.
+
+          For commands that switch the database, `db.namespace` SHOULD be set to the target database (even if the command fails).
+
+          It is RECOMMENDED to capture the value as provided by the application without attempting to do any case normalization.
         examples: ["instance1.products", "customers"]
 
+  - id: db.postgresql
+    type: span
+    extends: db.sql
+    brief: >
+      Attributes for PostgreSQL
+    attributes:
+      - ref: db.namespace
```

Comment:
now that we have sql-db-specific yamls, would you mind overwriting generic note/brief on `db.response.status_code` to be specific to that database?

https://github.com/open-telemetry/semantic-conventions/blob/87ec4e6e3b56effd895775f4f8f63e412bde7f75/model/database/spans.yaml#L385

I can also do it as a follow up - just let me know.

Reply:
I will add that 👍